pkg/controller/sparkapplication/sparkui.go

```diff
@@ -85,6 +88,12 @@ func createSparkUIIngress(app *v1beta2.SparkApplication, service SparkService, i
 			}},
 		},
 	}
+	if len(getIngressResourceAnnotations(app)) != 0 {
```

Comment:
Can you store the result of `getIngressResourceAnnotations` in a variable?

Reply:
done